rules/windows/defense_evasion_unusual_dir_ads.toml

```diff
@@ -22,8 +22,8 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where event.type in ("start", "process_started") and
-  process.args : "C:\\*:*"
+process where event.type == "start" and
+  process.args : ("C:\\*:*", "D:\\*:*", "E:\\*:*")  and process.args_count == 1
```

Comment:
```suggestion
  process.args : ("C:\\*:*", "D:\\*:*", "E:\\*:*") and process.args_count == 1
```

Reply:
We could even do `process.args : "?:\\*:*"` to cover all drive letters

Reply:
I like Brent's suggestion. There is also an extra space before `  and`